src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public class ColorSupplier extends Lottery {
+    public static String getRandomColor() {
+        Colors[] colors = Colors.values();
+        int randomIndex = new Random().nextInt(colors.length);
+        return colors[randomIndex].toString();
```

Comment:
Use `name()` instead of `toString()` to get the string representation of enum constants. This ensures that the correct name of the enum constant is returned, as recommended in the checklist .